.travis.yml

```diff
@@ -0,0 +1,41 @@
+# trusty beta image has jdk8, gcc4.8.4
+dist: trusty
+sudo: required
+# xcode8 has jdk8
+osx_image: xcode8
+# Not technically required but suppresses 'Ruby' in Job status message.
+language: java
+
+os:
+  - linux
+  - osx
+
+env:
+  - BAZEL=0.22.0
+
+before_install:
+  - |
+    if [ "${TRAVIS_OS_NAME}" = "osx" ]; then
+      OS=darwin
+    else
+      sysctl kernel.unprivileged_userns_clone=1
+      OS=linux
+    fi
+    wget -O install.sh "https://github.com/bazelbuild/bazel/releases/download/${BAZEL}/bazel-${BAZEL}-installer-${OS}-x86_64.sh"
+    chmod +x install.sh
+    ./install.sh --user
+    rm -f install.sh
+
+before_script:
+  - echo "startup --output_base=$HOME/.cache/bazel" >> .bazelrc
+  # Avoid progress indicators that can blow out log which makes using WEB UI difficult
+  - echo "common --noshow_progress" >> .bazelrc
+  - echo "common --noshow_loading_progress" >> .bazelrc
+  # This is so we understand failures better
+  - echo "build --verbose_failures" >> .bazelrc
+
+script:
+  - ./bazel_build_test.sh
```

Comment:
I dont think we need a list here. 
I can change that during the import process.